Makefile

```diff
@@ -58,12 +59,18 @@ unit-test:
 	go test -cover -coverprofile=coverage.txt ./...
 
 ### e2e-test:         Run e2e test cases (minikube is required)
-e2e-test: build-image-to-minikube
+e2e-test: build-image-to-minikube ginkgo-check
```

Comment:
We may run `ginkgo-check` before `build-image-to-minikube`, it would be het up if the e2e-test fails after the image was pushed to minikube node but the `ginkgo` tool is not found.